Makefile

```diff
@@ -0,0 +1,137 @@
+XDG_CACHE_HOME ?= $(HOME)/.cache
+export CACHE_DIR ?= $(XDG_CACHE_HOME)/zitadel-make
+
+LOGIN_DIR ?= ./
+LOGIN_BAKE_CLI ?= docker buildx bake
+LOGIN_BAKE_CLI_WITH_ARGS := $(LOGIN_BAKE_CLI) --file $(LOGIN_DIR)docker-bake.hcl --file $(LOGIN_DIR)apps/login-test-acceptance/docker-compose.yaml
+LOGIN_BAKE_CLI_ADDITIONAL_ARGS ?=
```

Comment:
Can you just defined the ARGS variable before the CLI_WITH_ARGS?

Reply:
No, becaise the order of the --file arguments matter

Reply:
Ah, reasonable.